### PR TITLE
Set Vault config vars in the environment for porklock containers

### PR DESCRIPTION
**Pull request by johnworth**
The VAULT_ADDR, VAULT_TOKEN, and JOB_UUID environment variables need to be set for porklock to read the iRODS config from Vault. This shouldn't affect any of the existing builds until I update the vendored version of dockerops in road-runner.

### CODE REVIEW

File: containers.go [hunk 1]
```diff
@@ -647,13 +647,25 @@ func (d *Docker) CreateDownloadContainer(job *model.Job, input *model.StepInput,
...
+	tag = d.cfg.GetString("porklock.tag")        // Used to pull the correct porklock image.
+	vaultaddress := d.cfg.GetString("vault.url") // Used by porklock to pull the iRODS config from Vault.
+	vaulttoken := d.cfg.GetString("vault.token") // Used by porklock to pull the iRODS config from Vault.
 
+	// The porklock image is needed before a container can be created from it.
 	if err = d.PorkPull(); err != nil {
 		return "", err
 	}
 
+	// It makes little sense to have only one of these set
```
Comment (ianmcorvidae) [inline containers.go:660]: Annoying Review Question time: would it make sense to have it outright cause an error if only one or the other is set, to help detect bad/probably wrong configurations?

**Review by ianmcorvidae — APPROVED** [review]
One probably-small question, otherwise seems pretty reasonable.

File: containers.go (hunk 1 above)
Comment (johnworth) [reply containers.go:660]: I've thought about that, too. Right now porklock will fail outright if one or the other isn't set, so the job will still fail. I'm undecided about whether or not that's better, but it's something we can change pretty easily if it becomes an issue in the future.

Comment (ianmcorvidae) [reply containers.go:660]: Sounds reasonable. Now that you mention it, I think porklock is the better place to have that error happen anyway. Closer to the real issue.

**Pull request merged**